src/Main/Java/cart/Cart.java

```diff
@@ -77,6 +77,9 @@ public void removeFromCart(Product product, int quantity) throws Exception {
     }
 
     public void addToCart(Product product, int quantity) throws Exception {
+        if (product.getSellPrice() <= 0) {
+            throw new Exception("Valore Negativo");
```

Comment:
Per avere un messaggio del genere dovremmo considerare solo i numeri minori di 0 , visto che prendiamo in esame anche casi in cui è uguale a 0 sarebbe meglio adattare il messaggio per rendendolo più esplicativo.